src/Illuminate/Support/Url.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Illuminate\Contracts\Support\Arrayable;
+
+class Url implements Arrayable
```

Comment:
Would love to see this implement `Stringable`, either explicitly or implicitly, and offer the ability to reconstruct the URL as a string.

Reply:
Done on both `Url` and `UrlQueryParameters`! 👍